examples/alternate-receive/alternate_receive.bal

```diff
@@ -0,0 +1,57 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/lang.runtime;
+
+type Response record {
+    record {
+        string 'worker;
+    } args;
+};
+
+function getFirstFetched(string url1, string url2) returns string? {
+    // workers `w1` and `w2` fetches the content from the respective URLs.
+    worker w1 {
+        string|error result = fetch(url1);
+        result -> w3;
+    }
+
+    worker w2 {
+        runtime:sleep(3);
+        string|error result = fetch(url2);
+        result -> w3;
+    }
+
+    worker w3 returns string? {
```

Comment:
Add a brief comment saying w3 waits for one to complete.

Reply:
Isn't that specified from the comment added here.

Reply:
Just a comment before the worker declaration. We can keep the inner comment for details.